mailpoet/tests/docker/codeception/docker-entrypoint.sh

```diff
@@ -169,6 +169,9 @@ wp theme install twentytwentyone --activate
 if [[ $MULTISITE == "1" ]]; then
   wp theme install twentytwentyone --url=$HTTP_HOST/$WP_TEST_MULTISITE_SLUG --activate
```

Comment:
I noticed that you tried upgrading to `twentytwentythree`. Why did you revert the change? Was there any blocker in upgrading?

Reply:
I realized that TT3 is also block-based theme. We have TT4 as a block theme so testing on TT1 will be non block theme. We can stay with TT1 and extend with TT4 and other themes.

Reply:
Thanks for the explanation!